### DISCUSSION TITLE
Does this Lib Support Multi-Device Auth and Session Saving.

### DISCUSSION THREAD

**Discussion by ghost** [Auties00/WhatsappWeb4j#121] [Q&A]
As we know thet whatsapp have launched a new feature which is MD (Multi Device) authentication . In this case a user can connect whasapp web to 4 devices and there is no need to keep whatsapp app online for using whatsapp web version which is Multi Device enabled account .  Some other libraries ( Whatsapp BOT libs )as baileys or whastsappp-web-js (Node.js) also have support to MD accounts . ... also how to save session because I dont want to Scan QR Code again and again.   + Is there any official Discord server or Community for this library ?

**Comment by OnlyCris** [comment]
this library is under update, the next update will exit very soon (like few days), it will implement the multidevice & session saving features support.
